tests/acceptance/expected-failures-with-ocis-server-owncloud-storage.md

```diff
@@ -422,3 +421,17 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUIResharing/reshareUsers.feature:336](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIResharing/reshareUsers.feature#L336)
 -   [webUIResharing/reshareUsers.feature:353](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIResharing/reshareUsers.feature#L353)
 -   [webUIResharing/reshareUsers.feature:365](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIResharing/reshareUsers.feature#L365)
+
+#### [Searching sharee with displayname](https://github.com/owncloud/ocis/issues/547)
```

Comment:
It would be nice if you could mention the PR that adds tests for this issue in the description of this PR. Anyway, the changes look good to me :+1: